x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/page.tsx

```diff
@@ -149,13 +149,13 @@ export const Page: FC<Props> = ({ jobId }) => {
                       {jobId === undefined && (
                         <FormattedMessage
                           id="xpack.ml.dataframe.analytics.creationPageTitle"
-                          defaultMessage="Create analytics job"
+                          defaultMessage="Create data frame analytics job"
```

Comment:
Wondering if we need the full `Create data frame analytics job` text here, or if just `Create job` is enough? Do the breadcrumbs provide sufficient context that we are creating a data frame analytics job?

In the anomaly detection jobs wizard for example, do we need it changed to `Create anomaly detection job: Advanced` or is it enough just to use `Create job: Advanced` as is used currently.

![image](https://user-images.githubusercontent.com/7405507/86032492-5c1f3600-ba2f-11ea-9e5c-49df06f3e24b.png)

Reply:
I think it's a good point that we have two options here: a) use the full name, which is what this PR currently proposes, or (b) use the really short name and align with what we do in anomaly detection. I think both of these options are valid, the conciseness of (b) is appealing.

Reply:
Awesome! Thanks for raising the question @peteharverson and for your insight @lcawl. My vote goes to the b option, too. I modify the PR accordingly.